File: serverfiles/framework_implementations/express/postController.js
const fs = require('fs');

exports.postImage = (req, res) => {
    const image = req.body.image.replace(/^data:image\/png;base64,/,"").toString("base64");
    res.status(200).json({
        res: 1
    })
}

exports.postObject = (req, res) => {
    res.status(200).json({
        res: 1
    })
}

exports.postSimple = (req, res) => {
    res.status(200).json({
        res: 1
    })
}


/* Dynamic */

exports.postImageDyn = (req, res) => {
    const image = req.body.image.replace(/^data:image\/png;base64,/,"").toString("base64");
    res.status(200).json({
        res: req.params.id
    })
}

exports.postObjectDyn = (req, res) => {
    res.status(200).json({
        res: req.params.id
    })
}

exports.postSimpleDyn = (req, res) => {
    res.status(200).json({
        res: req.params.id
    })
}

File: serverfiles/framework_implementations/aspnet/aspnet/Controllers/PostController.cs
using System;
using System.Text.Json;
using System.Text.Json.Serialization;
using Microsoft.AspNetCore.Mvc;

namespace aspnet.Controllers
{

    public class ImageBody
    {
        [BindProperty]
        public string Image { get; set; }
    }
    
    public class ObjectBody
    {
        [BindProperty]
        public string Lorum { get; set; }
    }

    public class PostController : ControllerBase
    {
        [HttpPost]
        [Route("post/image")]
        public IActionResult PostImage([FromBody] ImageBody imageBody)
        {
            var x = Convert.FromBase64CharArray(imageBody.Image.ToCharArray(), 0, imageBody.Image.Length);
            return new JsonResult(new {res=1});
        }
        
        [HttpPost]
        [Route("post/simple")]
        [Route("post/object")]
        public IActionResult PostObject([FromBody] ObjectBody objectBody)
        {
            return new JsonResult(new {res=1});
        }
        
        /* Dynamic */
        
        [HttpPost]
        [Route("post/image/{id}")]
        public IActionResult PostImageDyn([FromBody] ImageBody imageBody, int id)
        {
            var x = Convert.FromBase64CharArray(imageBody.Image.ToCharArray(), 0, imageBody.Image.Length);
            return new JsonResult(new {res=id});
        }
        
        [HttpPost]
        [Route("post/simple/{id}")]
        [Route("post/object/{id}")]
        public IActionResult PostObjectDyn([FromBody] ObjectBody objectBody, int id)
        {
            return new JsonResult(new {res=id});
        }
    }
}